website/source/docs/drivers/docker.html.md

```diff
@@ -614,8 +617,105 @@ user to the `docker` group so you can run Nomad without root:
+
+* `auth` stanza:
+    * `config`<a id="plugin_auth_file"></a> - Allows an operator to specify a
+  JSON file which is in the dockercfg format containing authentication
+  information for a private registry, from either (in order) `auths`,
+  `credHelpers` or `credsStore`. 
+    * `helper`<a id="plugin_auth_helper"></a> - Allows an operator to specify
+  a [credsStore](https://docs.docker.com/engine/reference/commandline/login/#credential-helper-protocol)
+  -like script on $PATH to lookup authentication information from external
+  sources. The script's name must begin with `docker-credential-` and this
+  option should include only the basename of the script, not the path.
+
+* `tls` stanza:
+    * `cert` - Path to the server's certificate file (`.pem`). Specify
+  this along with `key` and `ca` to use a TLS client to
+  connect to the docker daemon. `endpoint` must also be specified or
+  this setting will be ignored.
+    * `key` - Path to the client's private key (`.pem`). Specify this
+  along with `cert` and `ca` to use a TLS client to
+  connect to the docker daemon. `endpoint` must also be specified or
+  this setting will be ignored.
+    * `ca` - Path to the server's CA file (`.pem`). Specify this along
+  with `cert` and `key` to use a TLS client to connect to
+  the docker daemon. `endpoint` must also be specified or this setting
+  will be ignored.
+
+* `gc` stanza:
+    * `image` - Defaults to `true`. Changing this to `false` will
+  prevent Nomad from removing images from stopped tasks.
+    * `image_delay` - A time duration, as [defined
+  here](https://golang.org/pkg/time/#ParseDuration), that defaults to `3m`. The
+  delay controls how long Nomad will wait between an image being unused and
+  deleting it. If a tasks is received that uses the same image within the delay,
+  the image will be reused.
+    * `container` - Defaults to `true`. This option can be used to
+  disable Nomad from removing a container when the task exits. Under a name
+  conflict, Nomad may still remove the dead container.
+
+* `volumes` stanza:
+    * `enabled` - Defaults to `true`. Allows tasks to bind host paths
+  (`volumes`) inside their container and use volume drivers (`volume_driver`).
+  Binding relative paths is always allowed and will be resolved relative to the
+  allocation's directory.
+    * `selinuxlabel` - Allows the operator to set a SELinux
+  label to the allocation and task local bind-mounts to containers. If used
+  with `docker.volumes.enabled` set to false, the labels will still be applied
+  to the standard binds in the container.
+
 ## Client Configuration
 
+~> Note: client configuration options will soon be deprecated. Please use [plugin options][plugin-options] instead. See the [plugin stanza][plugin-stanza] documentation for more information.
```

Comment:
Another spot maybe we can just shorten it to `plugin stanza` as Plugin Options has very little content?

Reply:
Thanks for this tip, Michael. I implemented this suggestion in the main client configuration doc, but the plugin options links in the individual task drivers actually link back to more information/options, so I think we should keep it for at least those docs? If you take another pass at this section in particular and still feel we need to shorten, I'll be happy to make the change.

Reply:
Sounds good to me! :+1: